etc/redux-toolkit.api.md

```diff
@@ -167,20 +196,26 @@ export function getType<T extends string>(actionCreator: PayloadActionCreator<an
 // @public
 export function isPlain(val: any): boolean;
 
-// Warning: (ae-forgotten-export) The symbol "WithOptional" needs to be exported by the entry point index.d.ts
-// Warning: (ae-forgotten-export) The symbol "WithPayload" needs to be exported by the entry point index.d.ts
 // Warning: (ae-missing-release-tag) "PayloadAction" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public
-export type PayloadAction<P = void, T extends string = string, M = never, E = never> = WithOptional<M, E, WithPayload<P, Action<T>>>;
+export type PayloadAction<P = void, T extends string = string, M = never, E = never> = {

```

Comment:
This has to stay a type, as interfaces cannot include conditional logic (like adding and removing the property `meta` depending on the type value of `M`).

But I inlined the logic, so that TS resolves it into one readable type as soon as type parameters are known.